R/03-geojob-obj.R

```diff
@@ -56,6 +56,31 @@ setMethod("geojob", signature(), function(...) {
   return(geojob)
 })
 
+#'@param xml location of xml 
+#'@rdname geojob-methods
+#'@importFrom httr GET 
+#'@importFrom RCurl url.exists
+#'@importFrom XML xmlTreeParse xmlAttrs xmlRoot
+#'@aliases geojob,geojob-method
+setMethod("geojob", signature("ANY"), function(xml, urlSlot = NULL) {
```

Comment:
yes, a character method and an XML method would be great. and the char method would grab the doc and use the XML method